golden_marshmallows/schema.py

```diff
@@ -186,6 +184,7 @@ def generate_nested_fields(self, nested_map, new_fields):
                     snake_to_camel=self.snake_to_camel,
                     camel_to_snake=self.camel_to_snake,
                     many=val['many'],
+                    unknown=EXCLUDE,
```

Comment:
This along with lines 261-2 is where I wasn't really sure how to best deal with the nested removal.  This seems to work just fine based on the current tests.